cmd/ingress-operator/main.go

```diff
@@ -99,7 +100,7 @@ func main() {
 	}
 
 	// Set up the DNS manager.
-	dnsProvider, err := createDNSProvider(kubeClient, operatorConfig, dnsConfig, platformStatus)
+	dnsProvider, err := createDNSProvider(kubeClient, operatorConfig, dnsConfig, platformStatus, releaseVersion)
```

Comment:
Why add a new parameter instead of using `operatorConfig.OperatorReleaseVersion` as the logic for the other handlers does?

Reply:
Fixed